setup.py

```diff
@@ -51,8 +51,8 @@
     },
 
     install_requires=[
-        'pylint==1.4.5',
-        'pylint-django==0.6.1',
+        'pylint>=1.5.4,<2.0.0',
```

Comment:
We need 1.5.x for Python 3.5.  Hmm...how about I change this to `'pylint>=1.4.5,<2.0.0'`, and add `pylint==1.4.5` to edx-platform? That allows for edx-platform to remain stable, and other users to get the latest compatible version. We also don't have to touch this package again just to update Pylint.